deno.json

```diff
@@ -19,21 +19,27 @@
   },
   "compilerOptions": { "jsx": "react-jsx", "jsxImportSource": "preact" },
   "imports": {
+    "$fresh/": "https://deno.land/x/fresh@1.7.1/",
     "@/": "./",
-    "$fresh/": "https://raw.githubusercontent.com/denoland/fresh/60220dd33b5b0f6b5c72927c933dbc32a3c4734e/",
-    "preact": "https://esm.sh/preact@10.19.2",
-    "preact/": "https://esm.sh/preact@10.19.2/",
-    "preact-render-to-string": "https://esm.sh/*preact-render-to-string@6.2.2",
+    "@deno/gfm": "jsr:@deno/gfm@^0.9.0",
     "@preact/signals": "https://esm.sh/*@preact/signals@1.2.1",
     "@preact/signals-core": "https://esm.sh/*@preact/signals-core@1.5.0",
-    "tailwindcss": "npm:tailwindcss@3.4.1",
-    "tailwindcss/": "npm:/tailwindcss@3.4.1/",
-    "tailwindcss/plugin": "npm:/tailwindcss@3.4.1/plugin.js",
-    "$std/": "https://deno.land/std@0.208.0/",
-    "stripe": "npm:/stripe@13.5.0",
+    "@std/assert": "jsr:@std/assert@^1.0.4",
+    "@std/datetime": "jsr:@std/datetime@^0.225.2",
+    "@std/front-matter": "jsr:@std/front-matter@^1.0.4",
+    "@std/fs": "jsr:@std/fs@^1.0.3",
+    "@std/http": "jsr:@std/http@^1.0.5",
+    "@std/path": "jsr:@std/path@^1.0.4",
+    "@std/testing": "jsr:@std/testing@^1.0.2",
+    "@std/ulid": "jsr:@std/ulid@^1.0.0",
+    "@std/url": "jsr:@std/url@^0.225.1",
```

Comment:
not sure if this is generally preferred? I find jsr nicer tho personally. Happy to switch this back to `$std`.